### PR TITLE
Bug 1140321 - [RTL][Download] Fail to download the file, then the error ...

### PR DESCRIPTION
**Pull request by gmarty**
...icon is left-aligned and overlap with refresh icon in download list

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=02dfb7a53042)

File: apps/settings/style/downloads.css [hunk 1]
```diff
@@ -235,6 +235,11 @@
 }
 
 /* RTL rules */
+html:-moz-dir(rtl) #downloadList aside.download-status {
+  right: 0.2rem;
```
Comment (crh0716) [inline apps/settings/style/downloads.css]: As the icon is smaller than 30x30, it is aligned to the right. I would suggest to add `background-center` as you did in the previous patch. 
And `#downloadList aside` set `margin-right` to 0.5rem, so the value of right should be -0.3rem in rtl layout.

File: apps/settings/style/downloads.css [hunk 2]
```diff
@@ -235,6 +235,11 @@
...
+  left: auto;
```
Comment (crh0716) [inline apps/settings/style/downloads.css]: nit: Suggest to use `unset` if you want to override it as a default value.

File: apps/settings/style/downloads.css (hunk 1 above)
Comment (gmarty) [inline apps/settings/style/downloads.css]: That's overridden in line 192 `#downloadList ul > li[data-state="failed"] .download-status`. So I let it to 0.2rem for now.

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ac9f11319092)

File: apps/settings/style/downloads.css (hunk 1 above)
Comment (crh0716) [inline apps/settings/style/downloads.css]: It's fine to set `right` to 0.2rem, but we should also flip the margin of `#downloadList aside` in RTL mode (`margin: 1.75rem 0 0 0.5rem`).

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=633dcc85381a)

**Pull request merged**